indico/htdocs/sass/partials/_timelines.scss

```diff
@@ -166,3 +166,61 @@
         margin-top: 0;
     }
 }
+
+ul.deadline-timeline {
```

Comment:
what about we place it to _infogrids.scss ? Maybe not that related but maybe not bad either.